ci/ciimage/ubuntu-rolling/install.sh

```diff
@@ -50,6 +51,14 @@ dub_fetch dubtestproject
 dub build dubtestproject:test1 --compiler=ldc2
 dub build dubtestproject:test2 --compiler=ldc2
 
+# Remove debian version of Rust and install latest with rustup.
+# This is needed to get the cross toolchain as well.
+apt-get -y remove rustc || true
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup
+sh ./rustup -y
+sh ./rustup -y --target add x86_64-pc-windows-gnu
```

Comment:
The name chosen for the file is pretty confusing. You're running the rustup installer and not rustup itself.
While it does support installing additional targets via the `--target` parameter, I don't believe it supports being run again when Rust is already installed (which that first line does).

There's also a stray `add` in the second call.

You can either install everything in one call like this:
```
sh ./rustup -y --target x86_64-pc-windows-gnu
```

Or you can change the second line to an actual rustup invocation:
```
sh ./rustup -y
rustup target add x86_64-pc-windows-gnu
```

Reply:
I didn't realize the script we download is an installer, I thought it was rustup itself. Updated, thanks.